apps/core/components/ItineraryCard/ItineraryCard.js

```diff
@@ -107,5 +122,35 @@ const styles = StyleSheet.create({
     borderColor: defaultTokens.borderColorCard,
     borderBottomWidth: 1,
     borderTopWidth: 1,
+    web: {
+      alignSelf: 'center',
+      flexDirection: 'row',
+      alignItems: 'center',
+      width: '100%',
+      maxWidth: 720,
+      padding: parseInt(defaultTokens.spaceMedium, 10),
+      marginBottom: parseInt(defaultTokens.spaceMedium, 10),
+      borderBottomWidth: 0,
+      borderTopWidth: 0,
+      borderRadius: 3,
```

Comment:
You can use `borderRadiusNormal` from `designTokens`

Reply:
Thanks, I thought that there is only 2 and 6. Updated